libstuff/libstuff.cpp

```diff
@@ -663,7 +666,7 @@ int SParseHTTP(const char* buffer, size_t length, string& methodLine, STable& na
                 }
 
                 // If not processing a chunked body, then finish up.
-                else if (!SIEquals(nameValueMap["Transfer-Encoding"], "chunked")) {
+                else if (it == nameValueMap.end() || !SIEquals(it->second, "chunked")) {
```

Comment:
Why this change?

Reply:
Because this pollutes every request with a useless, empty `Transfer-Encoding` header currently. This just cleans that up.